python/cudf_polars/tests/dsl/test_serialization.py

```diff
@@ -0,0 +1,68 @@
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES.
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import pickle
+
+import pytest
+
+from polars.polars import _expr_nodes as pl_expr
+
+from cudf_polars.dsl.expressions.boolean import BooleanFunction
+from cudf_polars.dsl.expressions.datetime import TemporalFunction
+from cudf_polars.dsl.expressions.string import StringFunction
+
+
+@pytest.mark.parametrize(
+    "function", [BooleanFunction, TemporalFunction, StringFunction]
+)
```

Comment:
Let's abstract this into a reusable fixture:

```
@pytest.fixture(params=["BooleanFunction", "StringFunction", "TemporalFunction"])
def function(request):
    return request.param
```

Reply:
Good idea, done in https://github.com/rapidsai/cudf/pull/17418/commits/fcf820f0d39f046ce894bd1fc0876408aedb6177 . I've kept the module references instead of using strings and resolved them in the tests with `__name__`, let me know if you have a strong preference for strings instead.

Reply:
No, that's fine.